content/articles/data-representation-using-brython/index.md

```diff
@@ -0,0 +1,354 @@
+integrity="sha512-Dr8IR/2fk/pBmngFLduWN5Ys/t7lcPxGgLCulHdTdgDrDRnGb+ANMxGXq+DKVN87jom+N3BrzzI476Pf98RReA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/brython/3.10.4/brython_stdlib.js" integrity="sha512-kMRN6F4Yq4sNLbPG2lH3EO9n776JHHZub+UWogDxVjh9uTnoVo3wtN/rnQD4C4/AZtqI2zQdvdouGAAxOGwNeA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+```
+
+You can use any of the methods above to create your Brython project. After creating the Brython project, we can now create a piechart.
+
+#### Create a piechart using Brython.
+In this section, we will start by creating a piechart using Brython. To create the piechart, we will follow the following steps:
+
+##### Initializing the HTML5 boilerplate.
+To initialize the HTML5 boilerplate, use the following code in VisualStudio Code:
+
+```html
+<html lang="en">
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
+```
+
+##### Installing the Highcharts module.
+To install the Highcharts module, use this code:
+
+```npm
+npm install highcharts
+```
+
+> Note: If you used the Node Package Manager, make sure you link your script to the `exporting.js` file in the `node_module folder`.
+
+To link your script to the `exporting.js` file in the `node_module folder`, use this code:
+
+```html
+<!-- jquery -->
+<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
+<!-- exporting.js -->
+<script src="exporting.js"></script>
+```
+
+#####  Adding the `brython.js and brython_stdlib.js` file using our script tag. 
+
+First, we will create a file named `chart.py.`  We then import these files into our HTML file using the script tag and set the type to `"text/python3"`.
+
+- In the body section,  add the `onload` attribute `brython(1)`. It prints error messages on the browser's console. Within the body tag,  add a div tag with id `container`, the div has the inline styling below:
+
+```css
+#container{
+    min-width: 310px; 
+    height: 400px; 
+    margin: 0 auto
+}
+```
+
+The `html` code is shown below:
+
+```html
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <title>Piechart</title>
+
+        <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
+
+        <script src="./node_modules/highcharts/highcharts.js"></script>
+        <script src="exporting.js"></script>
+
+        <script type="text/javascript" src="./node_modules/brython/brython.js"></script>
+        <script src="./node_modules/brython/brython_stdlib.js"></script>
+        <script type="text/python3" src="chart.py"></script>
+
+    </head>
+    <body onload="brython(1)">
+
+        <div id="container" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
+
+    </body>
+</html>
+```
+
```

Comment:
Format this code snippet correctly by removing the unnecessary white spaces in the code.